pkgs/development/python-modules/pytest-testmon/default.nix

```diff
@@ -9,20 +9,18 @@
 buildPythonPackage rec {
   pname = "pytest-testmon";
   version = "0.9.16";
-  disabled = pythonOlder "3.5";
 
   src = fetchPypi {
     inherit pname version;
     sha256 = "df00594e55f8f8f826e0e345dc23863ebac066eb749f8229c515a0373669c5bb";
   };
 
-  doCheck = false; # tests written in Python 2; library compiles and runs properly otherwise.
-  buildInputs = with pkgs; [ glibcLocales ];
-  propagatedBuildInputs = with pythonPackages; [ pytest coverage ];
-  preConfigure = ''
-    export LC_ALL="en_US.UTF-8"
+  buildInputs = with pythonPackages; [ pytest coverage ];
```

Comment:
We should still propagate `coverage` though.

Reply:
Done, thanks!